packages/itwinui-react/src/core/Header/HeaderLogo.tsx

```diff
@@ -33,25 +38,25 @@ export type HeaderLogoProps = {
  * <HeaderLogo logo={<img src='image.png' />} />
  * <HeaderLogo logo={<img src='data:image/png;base64,...' />}>Downloaded Image</HeaderLogo>
  */
-export const HeaderLogo = (props: HeaderLogoProps) => {
-  const { className, children, logo, onClick, ...rest } = props;
-  const keyDownHandler = (e: React.KeyboardEvent<HTMLDivElement>) => {
-    if (
-      onClick &&
-      (e.key === 'Enter' || e.key === ' ' || e.key === 'Spacebar')
```

Comment:
Why removed keyboard functionality?

Reply:
because it is all handled automatically by `<button>`

Reply:
from PR description:
> removed some extra code that is not necessary - previously we were always rendering a <div> and doing extra work to turn it into button when onClick was passed.

you can try it in [storybook](https://itwin.github.io/iTwinUI/1109/react/?path=/story/core-header--full0. inspect the logo to see that it is a `<button>`, then try tabbing to it and press Enter or Space to see logs in Actions tab.